### PR TITLE
Adding ne_

### PR DESCRIPTION
**Pull request by soma2000-lang**
ne_ #10500

### CODE REVIEW

**Comment by soma2000-lang** [comment]
@ZoeCD just a friendly reminder.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 1]
```diff
@@ -4052,6 +4053,49 @@ def test_torch_instance_ne(
...
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        min_value=-1e04,
+        max_value=1e04,
+        allow_inf=False,
+    ),
+)
+def test_torch_instance_ne_(
+    dtype_and_x,
+    init_num_positional_args: pf.NumPositionalArgFn,
```
Comment (ZoeCD) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:4071]: Don't worry about the type hints, they are not needed here. Could you remove them please?

**Review by ZoeCD — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py (hunk 1 above)
Comment (soma2000-lang) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py]: @ZoeCD done!

**Comment by soma2000-lang** [comment]
@ZoeCD made the changes.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 2]
```diff
@@ -4052,6 +4052,49 @@ def test_torch_instance_ne(
...
+    dtype_and_x,
+    init_num_positional_args,
+    method_num_positional_args,
+    as_variable,
+    native_array,
+    frontend_method_data,
+    frontend,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_method(
```
[the remark this answers is not in the document]
Comment (soma2000-lang) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:4076]: @ZoeCD  we need to check the raw logs right ?

**Pull request closed without merging**